examples/fully-private-eks-cluster/vpc/backend.conf

```diff
@@ -0,0 +1,3 @@
+bucket = "terraform-ssp-github-actions-state" #Create a new S3 bucket for terraform state management and update this entry with the bucket details
+region = "us-west-2"
+key    = "private/vpc/terraform-main.tfstate"
```

Comment:
We don't want users to these values. Ask the users to enter the bucket and ket details and add comments